py/redrock/results.py

```diff
@@ -104,14 +108,22 @@ def read_zscan(filename):
                 - zwarn: 0=good, non-0 is a warning flag
 
     """
+    def encode_column(c):
+        return c.astype((str, c.dtype.itemsize))
```

Comment:
Is this a debugging leftover?  Why is this function here, replacing the one imported via `from .utils import encode_column`.  If we specifically need it here, please also copy the docstring.

Reply:
I honestly have no idea why I put this here (which was some time ago...). I'll remove it for this PR but if I figure out why I tripped on it in the first place I'll follow up!